docs/dataservice-write-example.md

```diff
@@ -109,6 +109,29 @@ Select the `dataservice-write-example` scheme from the schemes list in the `XCod
 
 Once everything is correctly set up, build and run the example application on your device and you should see the `Publish Successful` message in the main UI screen. If you encountered an error message, e.g. `Publish Failed`, please check the device's logs for the detailed description of the error.
```

Comment:
I suggest to rephrase as follows:

Once everything is set up correctly, build and run the example application on your device. After building and running of the example is complete, you see the `Publish Successful` message in the main UI screen. If you encounter an error message, e.g. `Publish Failed`, please check the device's logs for the detailed description of the error.

Reply:
These are not my changes and have been there even before my change. Could you please give comments on my changes?

Reply:
Looks good, no more comments from my side.

Reply:
Looks good, no more comments from my side.